### ISSUE TITLE
Windows installation support suggestion for installing ffmpeg

### ISSUE THREAD

**Issue by curryarts** [facebookresearch/demucs#314]
You may want to consider reminding windows users to open the Anaconda prompt window as an administrator when first installing ffmpeg. That is required for it to properly install.

___Start the Anaconda prompt _as an administrator_, and run the following bash:___

**Comment by CarlGao4** [comment]
Maybe it depends on which directory you are using, because my installation did not require to be ran as administrator.

**Comment by adefossez** [comment]
I agree I wouldn't recommend running as admin in general. I already had a sentence to say in case of permission error, run the prompt as administrator, I made it stand out more now: https://github.com/facebookresearch/demucs/blob/main/docs/windows.md#potential-errors
